### DISCUSSION TITLE
Removing duplicates in ASReview Lab

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1944] [Q&A]
hello, 

It seems like I was unable to remove all duplicates when preparing the dataset in EndNote, because, when uploading a file, ASReview lab states there are a number of duplicates. 

Is it possible to have ASReview Lab remove them, or set them aside?

**Comment by J535D165** [comment]
That's a good point. At the moment, you can only deduplicate outside of ASReview. We expect to release native deduplication in ASReview at the beginning of next year.

**Comment by george-gca** [comment]
You can use [ASReview Datatools deduplicate](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-dedup) function to do this before importing the dataset. Also, I've been working on a [solution](https://github.com/asreview/asreview-datatools/pull/52) to help find duplicates that are similar but not exactly the same, like when corrections have been issued for a paper.

**Comment by javimangal** [comment]
For R users: The ASySD R package can be of help. Here you can find some code and thoughts I wrote during this year's ASReview summerschool course https://github.com/javimangal/dedup-asreview
